lib/src/kbasesearchengine/events/handler/WorkspaceEventHandler.java

```diff
@@ -210,6 +212,34 @@ public static long parseDateToEpochMillis(final String workspaceFormattedDate) {
         return ws.getClient().administer(new UObject(command))
                 .asClassInstance(WS_INFO_TYPEREF);
     }
+
+
+    /** Get the workspace information for an object from the workspace service to which this
+     * handler is communicating.
+     * @param workspaceID the integer ID of the workspace.
+     * @param objectId the integer ID of the object.
+     * @return the object info as returned from the workspace.
+     * @throws IOException if an IO exception occurs.
+     * @throws JsonClientException if an error retrieving the data occurs.
+     */
+    public GetObjectInfo3Results getObjectInfo(
```

Comment:
follows getWorksSpaceInfo which is being used in other parts of the code.

Reply:
But this isn't right? So we might as well keep it private until it's needed IMO. Also IMO, the workspace method should probably return a class rather than a tuple and keep the tuples completely hidden from the rest of the code because they're so damn ugly, but in the interests of moving fast I didn't push for that. So again here, if this were public I'd want to translate the results into a less disgusting class, but since it's only internal we could just keep it private.

Reply:
Made it private. Yeah I agree in general it is either better to make it private or make a proper interface.